cmd/goose/driver_clickhouse.go

```diff
@@ -4,5 +4,5 @@
 package main
 
 import (
-	_ "github.com/ClickHouse/clickhouse-go"
+	_ "github.com/ClickHouse/clickhouse-go/v2"
```

Comment:
Do not change this in the current PR. Instead get the e2e tests working reliably and only then do a migration to `v2`

https://github.com/pressly/goose/pull/340